test/testData/config.json

```
@@ -125,6 +125,57 @@
                 ]
             }
         },
+        {
+            "testName": "Path-LBP-Given-In",
+            "chainId": "11155111",
+            "blockNumber": "8085514",
+            "currentTimestamp": "1744221012",
+            "swapPathInput": {
+                "swapKind": 0,
+                "amountRaw": "10000000",
+                "tokens": ["0xb19382073c7A0aDdbb56Ac6AF1808Fa49e377B75", "0xB77EB1A70A96fDAAeB31DB1b42F2b8b5846b2613"],
+                "pools": [
+                    {
+                        "poolAddress": "0x8E4Bd90c02b612594C96a31fADD7E4957bd5daBb",
+                        "poolType": "LIQUIDITY_BOOTSTRAPPING"
+                    }
+                ]
```

The latest PR changed the structure for these configs.
Basically it's able to accept multiple paths now, so you have to wrap this data into a `paths` array, like this:
```suggestion
                "paths":  [
                    "amountRaw": "10000000",
                    "tokens": ["0xb19382073c7A0aDdbb56Ac6AF1808Fa49e377B75", "0xB77EB1A70A96fDAAeB31DB1b42F2b8b5846b2613"],
                    "pools": [
                        {
                            "poolAddress": "0x8E4Bd90c02b612594C96a31fADD7E4957bd5daBb",
                            "poolType": "LIQUIDITY_BOOTSTRAPPING"
                        }
                    ]
                }
```

---

addressed and set debug to false.